vision/style_transfer/README.md

```diff
@@ -0,0 +1,73 @@
+# Fast Neural Style Transfer
+
+## Use-cases
+This artistic style transfer model mixes the content of an image with the style of another image. Examples of the styles can be seen [here](https://github.com/pytorch/examples/tree/master/fast_neural_style#models).
+
+
+## Description
+The model uses the method described in [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/pdf/1607.08022.pdf).
+  
+
+## Model
+ |Model        |Download  |MD5 Checksum| Download (with sample test data)|ONNX version|Opset version|
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|Mosaic|[6.6 MB](models/mosaic.onnx)  | 39f0d4d12cf758a7aa31eb150d66244a| [7.2 MB](models/mosaic.tar.gz)|1.4|9|
```

Comment:
@spandantiwari do these models need opset 9, or should we export them as opset 8 so that they are compatible with WinML too?

Reply:
@jennifererwangg let's add opset 8 models as a separate PR.